app/assets/javascripts/sign_up_async_request.js.erb

```diff
@@ -40,18 +41,9 @@ $( document ).ready(function() {
 					if (result.statusCode == 404) {
 
 						$(".spinner").css("display","none");
-
+						$('#user_email').attr('validation-passed',true)
```

Comment:
A linter would recommend adding a space after the comma.

Reply:
Removed this line.